backend/src/server.ts

```diff
@@ -0,0 +1,7 @@
+import app from './express';
+
+const port = process.env.PORT || 3000;
+
+app.listen(port, () => {
+  console.info(`Server running at http://localhost:${port}`);
```

Comment:
Locally this makes sense, but once you're running in OpenShift your host is going to change and its no longer going to be proper for keeping track! You can set an environment variable here to track your host :) 
Alternatively just write something like `Server running on port ${PORT}`

Reply:
Out of curiosity: Is there a simple, reliable way to get my 'host' and set it as en environment var? 

From what I found, it seems I could use 'os' and 'os.networkinterfaces'. Is that the easiest way to go if I really wanted to track my host?

I'll leave it as 'Server running on port ${API_PORT}' as suggested

Reply:
Nothing comes to mind, I would not deem it necessary. In OpenShift your URL will be static, and localhost is localhost. You'll have to target them properly from from the other containers. You can use environment variables like `NODE_ENV` to determind production or local builds and set routes / hosts that way. 

```mermaid
flowchart LR
    F(Frontend) -- Target API Local/Prod --> A(Express API)
    style A stroke:green,stroke-width:2px
    A -- Target Datase Local/Prod --> D(Postgres Database)
    D -- Results --> A
    A -- Server Response --> F
```